starter-code/src/components/Nav.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { dom } from "@fortawesome/fontawesome-svg-core";
+import { faHome } from "@fortawesome/free-solid-svg-icons";
+import { Link, NavLink } from "react-router-dom";
+import { FaHome } from "react-icons/fa";
+import "../App.css";
+
+// library.add(faHome);
+
+const Navbar = () => {
+  return (
+    <Link to="/">
+      <nav className="justify-content-center navbar navbar-dark bg-primary">
+        <div style={{ color: "white" }}>
```

Comment:
for small style try to use css, its good practice to stay away from inline styling as much as we can